test/test_helper.rb

```diff
@@ -1,5 +1,7 @@
 root_location = File.expand_path("../../", __FILE__)
 inc_dirs = Dir.glob("#{root_location}/library/*/src")
+# Needed to test Y2DIR support in Yast::Directory
+inc_dirs << "#{root_location}/library/general/test"
```

Comment:
ah, thats right :(
ok, so please at least use constant with explanation and use it like
```
# Y2DIRs location needed for testing purpose
ADDITIONAL_Y2DIRS = [
  # Needed to test Y2DIR support in Yast::Directory
  "#{root_location}/library/general/test"
]
```